app/CLI.hs

```diff
@@ -37,6 +42,8 @@ import           Cardano.Node.CLI
 data NodeCLIArguments = NodeCLIArguments {
     systemStart  :: !SystemStart
   , slotDuration :: !SlotLength
+  , genesisSpec  :: !(Last PartialGenesis)
+  , keyMaterialSpec :: !(Last PartialStaticKeyMaterial)
```

Comment:
All `::` used to be aligned; Doing that pollutes the git diff, but keeps things tidy when reading a module.

Reply:
Diff pollution was the reason I didn't do it, yeah.. Done.